### ISSUE TITLE
Cannot start boxes with vmware_desktop provider with VMware WS Pro 17 on Linux

### ISSUE THREAD

**Issue by Salvor42** [hashicorp/vagrant-vmware-desktop#102]
I have a licensed provider installed:

```
$ vagrant plugin list
vagrant-vmware-desktop (3.0.3, global)

$ dpkg -l | grep vagrant-vmware
ii  vagrant-vmware-utility                        1:1.0.22                           amd64        Vagrant utility for VMware Workstation and Player
```

But I cannot start any box:

```
$ vagrant up 
Bringing machine 'default' up with 'vmware_desktop' provider...
==> default: Box 'hashicorp/bionic64' could not be found. Attempting to find and install...
    default: Box Provider: vmware_desktop, vmware_fusion, vmware_workstation
    default: Box Version: >= 0
==> default: Loading metadata for box 'hashicorp/bionic64'
    default: URL: https://vagrantcloud.com/hashicorp/bionic64
==> default: Adding box 'hashicorp/bionic64' (v1.0.282) for provider: vmware_desktop
    default: Downloading: https://vagrantcloud.com/hashicorp/boxes/bionic64/versions/1.0.282/providers/vmware_desktop.box
==> default: Successfully added box 'hashicorp/bionic64' (v1.0.282) for 'vmware_desktop'!
==> default: Cloning VMware VM: 'hashicorp/bionic64'. This can take some time...
An error occurred while executing `vmrun`, a utility for controlling
VMware machines. The command and output are below:

Command: ["-T", "player", "snapshot", "/home/miesen/.vagrant.d/boxes/hashicorp-VAGRANTSLASH-bionic64/1.0.282/vmware_desktop/ubuntu-18.04-amd64.vmx", "3dd9736f-aa7a-4564-a9c2-e45c3e92fff7", {:notify=>[:stdout, :stderr]}]

Stdout: Error: The operation is not supported

Stderr: 
```

Of course, the player does not support snapshots (AFAIK). I tried `vmrum -T ws ...` by hand, which worked.
What can I do to prevent the player to be chosen?

#### Vagrant version

Vagrant 2.3.7

#### Host operating system

Debian GNU/Linux 11 (bullseye)

#### Vagrantfile

```ruby
Vagrant.configure("2") do |config|

  config.vm.box = "hashicorp/bionic64"

  config.vm.box_check_update = false

end

```

**Comment by chrisroberts** [comment]
Hi there,

Would you please provide a gist of the output from `vagrant up --debug`.

Thanks!

**Comment by Salvor42** [comment]
Of course:

https://gist.github.com/Salvor42/1837b06992a560d2375c0bf4d6ef1bd0

Thank you for your effort!

**Comment by eddiezab** [comment]
I just ran up into this. The solution (or I'd rather way a workaround) that worked for me is [here](https://github.com/hashicorp/vagrant-vmware-desktop/issues/91#issuecomment-1631463671).

My service file was is a different location, but the key part is the `-license-override professional`

**Comment by jovmarkovic** [comment]
`-license-override professional` Worked for me as well.

**Comment by ssplatt** [comment]
I'm trying to use this in Github Actions with Packer and a "standard" / "personal" license. Using the suggestions to force a license type does not fix the issue. Installing 3.0.2 or 3.0.1 does not fix the issue. 3.0.1 does act a little bit different in that it doesn't bomb out with an error, instead it hangs at "Calling Vagrant Up (this can take some time)..."

**Comment by ssplatt** [comment]
Does it sound like the root cause may be similar to https://github.com/hashicorp/packer-plugin-vmware/issues/229 ?

**Comment by chrisroberts** [comment]
@ssplatt is the host you are running on for github actions bare metal or does it support nested virtualization?

**Comment by chrisroberts** [comment]
And sorry for the late reply here. The output of:

```
/usr/lib/vmware/bin/vmware-vmx --query-license LicenseEdition
```

would be helpful in determining if the matching needs to be modified. Thanks!

**Comment by Salvor42** [comment]
Short comment: I'm no longer on topic, since I went to Proxmox in my homelab. But I'm happy if this thread leads to something useful! Thank you for your time!

**Comment by ssplatt** [comment]
> @ssplatt is the host you are running on for github actions bare metal or does it support nested virtualization?

yes, they do. I believe the problem I'm running into now is that the free license cannot be set without VMware trying to pop up a gui at first run. I have a line to `touch` a license file but that isn't getting rid of the first run workflow.

**Comment by yuravk** [comment]
I have the same issue, reported it here [ #91 ](https://github.com/hashicorp/vagrant-vmware-desktop/issues/91#issuecomment-2672466470) as well.
